dizoo/box2d/lunarlander/config/lunarlander_rnd_onppo_config.py

```diff
@@ -60,12 +55,13 @@
         type='lunarlander',
         import_names=['dizoo.box2d.lunarlander.envs.lunarlander_env'],
     ),
-    env_manager=dict(type='base'),
+    env_manager=dict(type='subprocess'),
     policy=dict(type='ppo'),
     reward_model=dict(type='rnd')
 )
 lunarlander_ppo_rnd_create_config = EasyDict(lunarlander_ppo_rnd_create_config)
 create_config = lunarlander_ppo_rnd_create_config
 
 if __name__ == "__main__":
+    from ding.entry import serial_pipeline_reward_model
```

Comment:
should be serial_pipeline_reward_model_onpolicy

Reply:
The main file function is fixed.